### PR TITLE
Round the face

### PR DESCRIPTION
**Pull request by carmen-tm**
I have improved the `main.css` style file to make the face round.

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-    position: absolute;
-    top: 50%;
-    transform: translate(-50%, -60%) rotate(90deg);
-  }
\ No newline at end of file
+  background: #fc0;
+  height: 100px;
+  position: relative;
+  width: 100px;
+  border-radius: 50%;
```
Comment (valerialp) [inline main.css]: muy bonito. :)

File: main.css [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+}
+.face {
+  font-family: 'courier new', serif;
+  font-size: 2.5em;
+  font-weight: bold;
```
Comment (valerialp) [inline main.css]: poco especifico. >:(

File: main.css (hunk 1 above)
Comment (carmen-tm) [reply main.css]: me encanta que te encante

File: main.css (hunk 2 above)
Comment (carmen-tm) [reply main.css]: ok I'm gonna changet it

**Pull request merged**